### ISSUE TITLE
[identification] base64 encoded data

### ISSUE THREAD

**Issue by mr-tz** [fireeye/flare-floss#281]
e.g. look for ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/= string

**Comment by RahulSankhla312** [comment]
Hello @mr-tz I started working on this issue. 

I have currently changed for stack_strings , that if a String is **base_64** , it is identified . I have attached the output . 
<img width="595" alt="image" src="https://github.com/mandiant/flare-floss/assets/97101245/7ea22acb-730a-469f-b992-92d6468e34ff">

Kindly see through it and suggest the changes necessary .

**Comment by williballenthin** [comment]
The intent of this issue is to track the idea that the "decoding routine identification phase" should take into account references to the Base64 alphabet. The intuition being that when the Base64 alphabet is seen, it's probably used nearby to decide some data. So in the same part of the code where we recognize non-zero xor and high xrefs, we should also inspect string references for the default Base64 alphabet.

**Comment by mr-tz** [comment]
sidebar: your idea here could be an interesting additional tag for [Quantumstrand](https://github.com/mandiant/flare-floss/tree/quantumstrand)

**Comment by vee1e** [comment]
@mr-tz good idea, i can work on this over the next few days to integrate into QS, both alphabet and strings. is this a priority enhancement?

**Comment by mr-tz** [comment]
not priority but a tag for this (or similar) would be neat
